Tests/TuistKitTests/Services/Inspect/InspectBuildCommandServiceTests.swift

```diff
@@ -24,10 +26,12 @@ struct InspectBuildCommandServiceTests {
     private let backgroundProcessRunner = MockBackgroundProcessRunning()
     private let dateService = MockDateServicing()
     private let serverURLService = MockServerURLServicing()
+    private let gitController = MockGitControlling()
 
     init() {
+        environment = Environment.mocked!
+        environment.reset()
```

Comment:
This is a big smell. Move the mocking to the test with `@Test(.withMockedEnvironment)` and refrain from resetting.